drupal/sites/default/boinc/modules/boincuser/boincuser.module

```diff
@@ -1241,9 +1246,14 @@ function join_page($type = null) {
   case 'new':
   default:
     $output .= '<ol>';
-    if (menu_valid_path(array('link_path' => $ruleslink))) {
+    if ( menu_valid_path(array('link_path' => $ruleslinkA)) ) {
       $output .= '  <li>' . bts("Read our !rules_and_policies.", array(
-        '!rules_and_policies' => l(bts('Rules and Policies', array(), NULL, 'boinc:join-page'), $ruleslink),
+        '!rules_and_policies' => l(bts('Rules and Policies', array(), NULL, 'boinc:join-page'), $ruleslinkA),
+      ), NULL, 'boinc:join-page') . '</li>';
+    }
+    else if (menu_valid_path(array('link_path' => $ruleslinkB))) {
+      $output .= '  <li>' . bts("Read our !rules_and_policies.", array(
+        '!rules_and_policies' => l(bts('Rules and Policies', array(), NULL, 'boinc:join-page'), $ruleslinkB),
```

Comment:
I don't like that there are now two identical `bts()` calls. What about rewriting it like this:
```
$ruleslink="";
if ( menu_valid_path(array('link_path' => $ruleslinkA)) ) {
  $ruleslink = $ruleslinkA;
} else if (menu_valid_path(array('link_path' => $ruleslinkB)) {
  $ruleslink = $ruleslinkB;
}
if ($ruleslink) {
output .= ... bts(...) ...;
}